metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_lineage_v2.py

```diff
@@ -288,16 +287,19 @@ def _populate_external_lineage_from_show_query(self, discovered_tables):
                 f"Populating external table lineage from Snowflake failed due to error {e}.",
             )
 
-    # Handles the case where a table is populated from an external location via copy.
+    # Handles the case where a table is populated from an external stage/s3 location via copy.
+    # Eg: copy into category_english from @external_s3_stage;
     # Eg: copy into category_english from 's3://acryl-snow-demo-olist/olist_raw_data/category_english'credentials=(aws_key_id='...' aws_secret_key='...')  pattern='.*.csv';
-    def _populate_external_lineage_from_access_history(
+    # NOTE: Snowflake does not log this information to the access_history table.
+    def _populate_external_lineage_from_copy_history(
         self, discovered_tables: List[str]
     ) -> None:
-        query: str = SnowflakeQuery.external_table_lineage_history(
```

Comment:
Can you add a comment on this method that it should be removed once legacy lineage is removed?

Reply:
sure